src/org/zaproxy/zap/extension/fuzz/httpfuzzer/ui/HttpFuzzerResultsTableModel.java

```diff
@@ -262,6 +263,19 @@ public String getType() {
         public Map<String, Object> getCustomStates() {
             return customStates;
         }
+        /** 
+         * Represents the 'FuzzResultTableEntry' as a string with all values separated by a comma. Is called by
+         * the Export button's actionListener in 'HttpFuzzResultsContentPanel'. 
+         * 
+         * A HttpFuzzerReflectionDetectorStateHighlighter is used to reveal true value of the customState field
+         */
+        public String toString() {

```

Comment:
Did you want me to add all the fields as well? The original issue asked to have the same values from the results table -- of course I can add all those fields as well. Instead of overriding the toString method, I can simply create a helper method to format the data.